client.go

```diff
@@ -1,10 +1,10 @@
 // Copyright (c) David Bond, Tailscale Inc, & Contributors
 // SPDX-License-Identifier: MIT
 
-// Package tsclient contains a basic implementation of a client for the Tailscale HTTP API.
+// package tailscale contains a basic implementation of a client for the Tailscale HTTP API.
```

Comment:
If our goal is to advertise this as the preferred standard client, maybe we should make this more explicit:
```suggestion
// Package tailscale implements a client for the Tailscale HTTP API.
```

Nit: Please capitalize "Package".